openoa/utils/plot.py

```diff
@@ -1932,3 +1932,284 @@ def plot_wake_losses(
+        N_col = 2
+    elif len(ws_bins) >= 13:
+        figure_kwargs.setdefault("figsize", (16, 17))
+        fig, axs = plt.subplots(
+            int(np.floor((len(ws_bins) - 1) / 3) + 1), 3, sharex=True, **figure_kwargs
+        )
+        N_col = 3
+    elif len(ws_bins) >= 10:
+        figure_kwargs.setdefault("figsize", (16, 14))
+        fig, axs = plt.subplots(4, 3, sharex=True, **figure_kwargs)
+        N_col = 3
+    elif len(ws_bins) >= 7:
+        figure_kwargs.setdefault("figsize", (16, 11))
+        fig, axs = plt.subplots(3, 3, sharex=True, **figure_kwargs)
+        N_col = 3
+    elif len(ws_bins) >= 5:
+        figure_kwargs.setdefault("figsize", (16, 7.5))
+        fig, axs = plt.subplots(2, 3, sharex=True, **figure_kwargs)
+        N_col = 3
+
+    for i, ws in enumerate(ws_bins):
+        if UQ:
+            norm_factor = curve_fit_params_ws[:, i, 0].mean()
+
+            y_min = np.nanmin(np.nanpercentile(power_values_vane_ws[:, i, :], 2.5, 0))
+            y_max = np.nanmax(np.nanpercentile(power_values_vane_ws[:, i, :], 97.5, 0))
+
+            axs[int(np.floor(i / N_col))][i % N_col].fill_between(
+                vane_bins,
+                np.percentile(power_values_vane_ws[:, i, :], 2.5, 0) / norm_factor,
+                np.percentile(power_values_vane_ws[:, i, :], 97.5, 0) / norm_factor,
+                color=power_color_code,
+                label="_nolegend_",
+                **plot_kwargs_fill,
+            )
+            axs[int(np.floor(i / N_col))][i % N_col].scatter(
+                vane_bins,
+                np.mean(power_values_vane_ws[:, i, :], 0) / norm_factor,
+                color=power_color_code,
+            )
+            axs[int(np.floor(i / N_col))][i % N_col].plot(
+                vane_bins,
+                cos_curve(
+                    vane_bins,
+                    curve_fit_params_ws[:, i, 0].mean() / norm_factor,
+                    curve_fit_params_ws[:, i, 1].mean(),
+                    curve_fit_params_ws[:, i, 2].mean(),
+                ),
+                color=curve_fit_color_code,
+                label="_nolabel_",
+            )
+            axs[int(np.floor(i / N_col))][i % N_col].plot(
+                2 * [curve_fit_params_ws[:, i, 1].mean()],
+                [
+                    0.01 * np.floor(y_min / norm_factor / 0.01),
+                    0.01 * np.ceil(y_max / norm_factor / 0.01),
+                ],
+                color=curve_fit_color_code,
+                linestyle="--",
+                label=f"Max. Power Vane Angle = {round(curve_fit_params_ws[:,i,1].mean(),1)}$^\circ$",
+            )
+
+            axs[int(np.floor(i / N_col))][i % N_col].set_title(
+                f"{ws} m/s\nYaw Misalignment = {np.round(np.mean(yaw_misalignment_ws[:,i]),1)}$^\circ$ [{np.round(np.percentile(yaw_misalignment_ws[:,i],2.5),1)}$^\circ$, {np.round(np.percentile(yaw_misalignment_ws[:,i],97.5),1)}$^\circ$]"
```

Comment:
I'm not normally one to make single-use variables, but I think it might be best to compute the numbers that go in the f-string just above this line because it's so long. Additionally, parentheses can be used to write strings across multiple lines like so:

```python
ax.set_title(
    (
        f"{ws} m/s\nYaw Misalignment = "
        f"{np.round(np.mean(yaw_misalignment_ws[:,i]),1)}$^\circ$ "
        f"[{np.round(np.percentile(yaw_misalignment_ws[:,i],2.5),1)}$^\circ$, "
        f"{np.round(np.percentile(yaw_misalignment_ws[:,i],97.5),1)}$^\circ$]"
    )
)
```